src/components/form/fields/schema-field.ts

```diff
@@ -0,0 +1,157 @@
+import { LimeElementsAdapter } from '../adapters';
+import JSONSchemaField from 'react-jsonschema-form/lib/components/fields/SchemaField';
+import React from 'react';
+import { FieldProps } from './types';
+import { isEmpty, capitalize } from 'lodash-es';
+
+const hasCustomComponent = (schema): boolean => {
+    return Boolean(schema.lime?.component?.name);
+};
```

Comment:
I'm not sure if we should have some error handling here or just ignore the custom rendering if the component does not exist :thinking: 
We can check the existance of a component by calling `customElements.get(schema.lime?.component?.name)`

Reply:
hmm ya good idea i would go with throwing an error i think. silent failures are awful to debug.